scrapi/harvesters/biomedcentral.py

```diff
@@ -6,134 +6,128 @@
 Example API query: http://www.biomedcentral.com/search/results?terms=*&format=json&drpAddedInLast=1&itemsPerPage=250#2015-04-23
 """
 
+
 from __future__ import unicode_literals
 
 import json
 import logging
-from datetime import date, timedelta, datetime
-
-from nameparser import HumanName
+from furl import furl
+from datetime import date, timedelta
 
 from scrapi import requests
 from scrapi import settings
 from scrapi.base import JSONHarvester
 from scrapi.linter.document import RawDocument
-from scrapi.base.helpers import build_properties, datetime_formatter
+from scrapi.base.helpers import build_properties, datetime_formatter, compose, default_name_parser
 
 logger = logging.getLogger(__name__)
 
 
-def process_contributors(authors):
+def process_urls(urls):
+    all_uris = [url['value'] for url in urls]
 
-    authors = authors.strip().replace('<span class="author-names">', '').replace('</span>', '')
-    authors = authors.split(',')
+    processed_uris = {
+        'canonicalUri': all_uris[0],
```

Comment:
Can you fix this so that the DOI given shows up as the canonicalUri? If you check out the results, the canonical URL should lead to the biomedcentral splash page.